apps/src/templates/levelSummary/summary-entry-point.module.scss

```diff
@@ -0,0 +1,34 @@
+@import "color.scss";
+
+.summaryEntryPoint {
+  padding-bottom: 8px;
+}
+
+.responseCounter {
+  display: inline-block;
+  padding: 0 12px;
+
+  height: 30px; // button height;
+  line-height: 30px;
+
+  p {
+    margin: 0;
+  }
+
+  span.counter {
+    padding: 0 8px;
+    font-weight: bold;
+  }
+
+  span.text {
+    display: block;
+  }
+
+  /* stylelint-disable selector-pseudo-class-no-unknown */
```

Comment:
just for my understanding: why is this needed?

Reply:
it's for the `:global(.fa)` selector, because the `fa` class doesn't come from inside this css module and would otherwise get a generated name that wouldn't match the element

Reply:
gotcha thanks!